linux_os/guide/system/software/integrity/fips/package_dracut-fips-aesni_installed/anaconda/shared.anaconda

```diff
@@ -0,0 +1,3 @@
+# platform = Red Hat Enterprise Linux 6,Red Hat Enterprise Linux 7,Oracle Linux 7
+
+package --add=dracut-fips-aesni
```

Comment:
The reporter in the issues #3221 says that:

> We cannot install it indiscriminately, as supported configuration for Red Hat Enterprise Linux 7 is this package installed only in case CPU supports given instruction set. So if we want to install package, we need to first check the support.

> That means we cannot (at least without changes to anaconda addon) install it during installation whatsoever. 

From what I get Anaconda still doesn't have any changes like that.

My understanding is that this remediation will break installation on other architectures. I haven't tested it, though.

Reply:
@jan-cerny went ahead and removed the anaconda remediation for now.